src/components/Forum/Forum.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+    AppBar,
+    Avatar,
+    Button,
+    Container,
+    Divider,
+    ListItem,
+    ListItemAvatar,
+    ListItemText,
+    Toolbar,
+    Typography,
+} from "@mui/material";
+import React, { useState } from "react";
+import TextareaAutosize from "@mui/base/TextareaAutosize";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import ReplyIcon from "@mui/icons-material/Reply";
+import SendIcon from "@mui/icons-material/Send";
+import {
+    ConversationContainer,
+    DescriptionContainer,
+    MainContainer,
+    Reply,
+} from "./Forum.styles";
+
+export const Forum: React.FC = () => {
+    const [replyButton, setreplyButton] = useState(false);
+    const openDescription = () => {
+        setreplyButton(true);
+    };
+    return (
+        <MainContainer>
+            <AppBar position="static">
+                <Container maxWidth="xl">
+                    <Toolbar disableGutters>
+                        <Typography variant="h6">Forum</Typography>
+                    </Toolbar>
+                </Container>
+            </AppBar>
+            <br />
+            <Typography variant="h6">Description</Typography>
```

Comment:
All the constants should be moved inside either UIConstants or the Coding Constants based on the usage.

Reply:
moved the constants to UI constants